### ISSUE TITLE
Search results dialog isn't refreshed after removing files

### ISSUE THREAD

**Issue by fulalas** [lxde/pcmanfm-qt#491]
Do a search, delete some files inside the result and they'll keep showing.

**Comment by tsujan** [comment]
Yes, it's an old bug that I've always forgotten after reloading the folder ;) Will look into it.

**Comment by fulalas** [comment]
I just want to tell that you're doing a great work managing all these issues. It's nice to have this close channel between users and developers :)

**Comment by fulalas** [comment]
Ah, just to make the report more precise: the bug also happens when renaming.

**Comment by tsujan** [comment]
Self-talk: The cause is simple: `GFileMonitor` can't be created for the search folder. Whether this is a bug in `folder.cpp` or a faulty design of search, I don't know yet.

**Comment by tsujan** [comment]
Actually, `GFileMonitor` can't be created in other cases either, for example, with SFTP. It seems nothing can be done in such cases. However, further investigation is required.

**Comment by simplelinuxuser** [comment]
Other file managers using gvfs do this though - i tried with gnome files and after deleting a file with pcmanfm it is immediately removed there
